scripts/gee

```diff
@@ -218,9 +222,10 @@ readonly GEE_BUILDLOGS_PROJECT="${GEE_BUILDLOGS_PROJECT:-cloud-build-290921}"
 readonly TRUE=0
 readonly FALSE=1
 readonly GEE_ON_ASTORE="test/gee"
-readonly GIT=/usr/bin/git
-readonly GH=/usr/bin/gh
-readonly JQ=/usr/bin/jq
+readonly GIT_PATHS=(/usr/enfabrica/bin/git ~/bin/git /usr/bin/git)
```

Comment:
@jonathan-enf Can you also add `/enf_mounts/bin/git`, `/enf_mounts/bin/gh`, `/enf_mounts/bin/jq`, `/enf_mounts/bin/enkit` to the search path? This is another place that will have possible tool versions.

Reply:
done